webpack.config.js

```diff
@@ -272,7 +272,7 @@ function* webpackConfig( env, argv ) {
 		},
 		plugins: [
 			new ProvidePlugin( {
-				React: 'react',
+				React: '@wordpress/element',
```

Comment:
Is this needed? I removed it and everything still built fine. It's minor, and I know it _is_ needed in the Jest tests and in Storybook, but maybe we should remove it here.

Reply:
Nevermind! I see why it's required now; all good 👍🏻

Reply:
Yes, it is needed for e2e and VRT tests 👍